subsys/bluetooth/services/Kconfig.dis

```diff
@@ -7,9 +7,24 @@
 
 menuconfig BT_GATT_DIS
 	bool "Enable GATT Device Information service"
+	default n
 
 if BT_GATT_DIS
 
+config BT_GATT_DIS_DYNAMIC
+	bool "Enable Dynamic data in Device Information Service"
+	help
+	 Enable Dynamic data in Device Information Service.
```

Comment:
Im not quite happy about the use of Dynamic term here, it perhaps lead to the impression that it can be changed at runtime or that it is writeable, maybe Options would be better here?

Reply:
Copied the term from the device name that can be changed runtime. So no problem changing it to settings.

Reply:
Changed